tfexec/internal/e2etest/show_test.go

```diff
@@ -67,8 +66,8 @@ func TestShow(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		if !reflect.DeepEqual(actual, &expected) {
-			t.Fatalf("actual: %s\nexpected: %s", spew.Sdump(actual), spew.Sdump(expected))
+		if diff := cmp.Diff(expected, actual, cmpopts.IgnoreFields(tfjson.State{}, "TerraformVersion")); diff != "" {
```

Comment:
you should wrap this in a `compareState` func or something to ensure all usages are the same